docs/_ext/traverse_operators_sensors.py

```diff
@@ -17,7 +17,7 @@
 CURRENT_FILE_PATH = Path(__file__)
 ASTRONOMER_PROVIDERS_PATH = CURRENT_FILE_PATH.parent.parent.parent / "astronomer" / "providers"
 
-BASE_CLASS_DEF_URL = "/astronomer-providers/docs/_build/html/_api/"
+BASE_CLASS_DEF_URL = "/en/stable/_api/"
```

Comment:
Is there a way we could dynamically get the base URL? Otherwise this would point to "stable" docs even for "latest".

Example: https://astronomer-providers.readthedocs.io/en/latest/providers/operators_and_sensors_list.html

Reply:
Created another PR to address the change needed to use the right version of the document: https://github.com/astronomer/astronomer-providers/pull/569

We're in need of using javascript here until we move everything to native Sphinx with the work on this ticket: https://app.zenhub.com/workspaces/airflow-providers-team-61a89ee7fbf048001dc8378d/issues/astronomer/astronomer-providers/508